### PR TITLE
Apply `black` to `astropy.config`

### LINKED ISSUE #13946: Apply `black` to `astropy.config`
Apply `black` to `astropy.config` according to the process laid out by [APE 20](https://github.com/astropy/astropy-APEs/blob/main/APE20.rst).

### PR DESCRIPTION
**Pull request by WilliamJamieson**
### Description

Apply `black` to `astropy.config` according to the process laid out by [APE 20](https://github.com/astropy/astropy-APEs/blob/main/APE20.rst).

Resolves #13946

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Do the proposed changes actually accomplish desired goals?
- [ ] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [ ] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [ ] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [ ] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [ ] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label. Codestyle issues can be fixed by the [bot](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html#pre-commit).
- [ ] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [ ] Is this a big PR that makes a "What's new?" entry worthwhile and if so, is (1) a "what's new" entry included in this PR and (2) the "whatsnew-needed" label applied?
- [ ] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [ ] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

**Review by nstarman — APPROVED** [review]

**Comment by WilliamJamieson** [comment]
@nstarman if you think this PR is ready for string normalization, please move it to the "Ready for string normalization" column in https://github.com/orgs/astropy/projects/10.

**Comment by WilliamJamieson** [comment]
> @nstarman if you think this PR is ready for string normalization, please move it to the "Ready for string normalization" column in [github.com/orgs/astropy/projects/10](https://github.com/orgs/astropy/projects/10).

Thanks for moving this forward. I have now normalized the strings. If things look good please move this to the "Ready for Squash" column.

**Comment by WilliamJamieson** [comment]
> Rebase and push? I can't merge currently.

Right now only the 5.2 release managers can merge.

File: astropy/config/tests/test_configs.py [hunk 1]
```diff
@@ -99,67 +99,71 @@ def test_set_temp_cache_resets_on_exception(tmp_path):
...
-    assert '# log_exceptions = False' in conf
+    assert "# unicode_output = False" in conf
+    assert "[io.fits]" in conf
+    assert "[table]" in conf
+    assert "# replace_warnings = ," in conf
+    assert "[table.jsviewer]" in conf
+    assert (
+        "# css_urls = https://cdn.datatables.net/1.10.12/css/jquery.dataTables.css,"
+        in conf
+    )
```
Comment (eteq) [inline astropy/config/tests/test_configs.py:152]: I found this particular case *very* hard to understand/read with the multi-line form. Are we willing to skip this one (@nstarman , if you strongly disagree with me I'm ok with backing down)
```suggestion
    assert "# css_urls = https://cdn.datatables.net/1.10.12/css/jquery.dataTables.css," in conf  # fmt: skip
```

**Review by eteq — APPROVED** [review]
I am happy with this too except for *one* change I just noticed while reading through the whole block, which I suggested inline to back out and skip black on the one line.  @nstarman may want to have an opinion on that one.

Other than that it looks good to me too!

**Pull request merged**